spec/models/user_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+require 'rails_helper'
+
+describe User do
+
+  it { should have_many :albums }
+
+
```

Comment:
should test User validations as well as associations! You might want to make sure your User model validations also reflect your database validations